spec/rubocop/cop/rspec/describe_method_spec.rb

```diff
@@ -27,7 +25,7 @@
 
   it 'skips specs not having a string second argument' do
     expect_no_offenses(<<-RUBY)
-      describe Some::Class, :config do
```

Comment:
Please keep this, it's a part of the example. Another symbol will do, e.g. `:foo`.

Reply:
Oops, removed with semi-auto cleanup 😅